docs/make.jl

```diff
@@ -21,11 +21,11 @@ if "--quarto" ∈ ARGS
     using CondaPkg
     CondaPkg.withenv() do
         @info "Rendering Quarto"
-        tutorials_folder = (@__DIR__) * "/../tutorials"
-        # instantiate the tutorials environment if necessary
-        Pkg.activate(tutorials_folder)
-        Pkg.resolve()
-        Pkg.instantiate()
+        # tutorials_folder = (@__DIR__) * "/../tutorials"
+        # # instantiate the tutorials environment if necessary
+        # Pkg.activate(tutorials_folder)
+        # Pkg.resolve()
```

Comment:
This will not help much and just lead to tutorials failing to start every now and then; every tutorial runs in its own Julia call and every tutorial starts with activating this environment, so not updating/resolving it – led to every now and then Python not finding Julia.

Reply:
But I have to be honest, I do not fully understand all the resolving that is done here, so if you see a better way feel free to improve this. I noticed, these lines are necessary.

Reply:
I was hoping it would just go back to the environment used for docs but as I said, I don't really understand all this stuff.

Reply:
Every quarto is run in a separate Julia call and every notebook in the very first cell activates the tutorials environment.

Reply:
OK, so I won't touch the tutorials that still work.